.gitignore

```diff
@@ -1,2 +1,5 @@
 dist.js
+.DS_Store
```

Comment:
you could add `.DS_Store` to your global `~/.gitignore`. but it's probably good to keep this line in here for future contributors.